pkg/controller/installation/bootstrapReconciler.go

```diff
@@ -110,6 +120,24 @@ func (r *Reconciler) checkCloudResourcesConfig(ctx context.Context, serverClient
 	return integreatlyv1alpha1.PhaseCompleted, nil
 }
 
+func (r *Reconciler) reconcilerRHMIConfigCR(ctx context.Context, serverClient k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
+	rhmiConfig := &integreatlyv1alpha1.RHMIConfig{
+		TypeMeta: metav1.TypeMeta{},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "rhmi-config",
+			Namespace: RHMICustomerConfigNamespace,
```

Comment:
Just a question, is this always going to be in the RHMI operator namespace? If it is, would probably be better to use `r.ConfigManager.GetOperatorNamespace()` instead of defining the namespace, otherwise do ignore :thinking:

Reply:
Yes, I think it will remain in the RHMI operator namespace. I'll change that so. @KevFan